scripts/deploy.sh

```diff
@@ -22,15 +24,21 @@ case "${env}" in
 esac
 
 FILE_PATHS=(
-  "packages/logs/bundle/datadog-logs-eu.js"
-  "packages/logs/bundle/datadog-logs-us.js"
   "packages/logs/bundle/datadog-logs.js"
-  "packages/rum-recorder/bundle/datadog-rum-recorder.js"
-  "packages/rum/bundle/datadog-rum-eu.js"
-  "packages/rum/bundle/datadog-rum-us.js"
   "packages/rum/bundle/datadog-rum.js"
+  "packages/rum-recorder/bundle/datadog-rum-recorder.js"
 )
 
+# legacy files do not support suffix
+if [[ -z $suffix ]]; then
+  FILE_PATHS+=(
+    "packages/logs/bundle/datadog-logs-eu.js"
+    "packages/logs/bundle/datadog-logs-us.js"
+    "packages/rum/bundle/datadog-rum-eu.js"
+    "packages/rum/bundle/datadog-rum-us.js"
+  )
+fi
```

Comment:
Do we still update these files then?

Reply:
I found no data to check if they are still used by users. 
To remove them, would mean to stop maintaining them. This decision implies to have a deprecation plan which, I think, is out of the scope of this PR.

Reply:
My question was probably not clear enough. 
FMU of the changes, these files are updated only if suffix is not provided to the deploy script but I have not seen these kind of calls.
am I missing something or does this change stop to update these files?

Reply:
fixed by 36edccbf36d0181c2cca6c038f8d1acb401aeb1b

Reply:
yes I see ;) Indeed